src/amp-react-layout.js

```diff
@@ -153,6 +161,17 @@ export function AmpWithLayout(props) {
     // TODO: partial implementation.
   }
 
+  // Placeholder.
+  if (placeholder && !hasBeenLoaded) {
+    // TBD: how to force placeholder to take 100% space?
+    layoutChildren.push(placeholder);
+  }
+
+  // Loading indicator.
+  if (!hasBeenLoaded) {
```

Comment:
Does loading indicator go over the placeholder?

Reply:
Yes. Currently that's the case in AMP and I think that's expected in all cases.